src/tests/DataFiles/sigma_cda/c_ref.mm

```diff
@@ -0,0 +1,17 @@
+0.000576252
+-0.000250517
+-0.000250509
+-3.1571e-05
+-0.00319127
+-0.0340969
+-0.0340969
+ 0.540951
+-0.0498701
+-0.161544
+-0.161544
+-0.342083
+-0.747893
+-0.655768
+-0.655768
+-0.389623
+-0.0743816
```

Comment:
Wrong file format.

Reply:
What is the right format exactly? I see this needs the header lines for example. But is it described anywhere?